app/controllers/looks_controller.rb

```diff
@@ -36,6 +36,24 @@ def destroy
         redirect_to looks_path
     end
 
+    def edit
+        @look = Look.find(params[:id])
+    end
+
+    def update
+        @look = Look.find(params[:id])
+        if params[:look][:photos] == [""]
+            params[:look].delete(:photos)
+        end
+        respond_to do |format|
+            if @look.update(look_params)
```

Comment:
Is there a reason not to raise ActiveRecord validations here?
Remember to always prefer the versions of save, create and update methods ended with the bang character `!` 

```suggestion
            if @look.update!(look_params)
```